modules/vertx-swagger-codegen/src/main/java/com/github/phiz71/vertx/swagger/codegen/JavaVertXServerGenerator.java

```diff
@@ -131,12 +131,14 @@ public String getHelp() {
     public void processOpts() {
         super.processOpts();
 
-        if (additionalProperties.containsKey(FUTURE_GENERATION_OPTION)) {
+        if (additionalProperties.containsKey(RX_INTERFACE_OPTION)) {
             apiTemplateFiles.clear();
-            apiTemplateFiles.put("apiFuture.mustache", // the template to use
+            apiTemplateFiles.put("apiRX.mustache", // the template to use
```

Comment:
Looks good, although I am wondering how well this approach of multiple mustache files (that might actually share several things) will scale in the future. For example, I have a feature in my fork where I generate something like `@Inject PetApi service` instead of `PetApi service = new PetApiImpl()`, resulting in a file "injectVerticle.mustache". This will make it hard to maintain in the future.

Is there a way to modularize the template files more?

I think it is fine how we have it currently, but just wanted to raise it so we can think about it.